x-pack/plugins/observability_solution/infra/public/components/asset_details/constants.ts

```diff
@@ -15,5 +15,7 @@ export const APM_HOST_FILTER_FIELD = 'host.hostname';
 export const ASSET_DETAILS_URL_STATE_KEY = 'assetDetails';
 
 export const INTEGRATIONS = {
-  [INTEGRATION_NAME.kubernetes]: 'kubernetes.node',
+  [INTEGRATION_NAME.kubernetesNode]: 'kubernetes.node',
+  [INTEGRATION_NAME.kubernetesContainer]: 'kubernetes.container',
+  [INTEGRATION_NAME.dockerContainer]: 'docker.container',
```

Comment:
```suggestion
  [INTEGRATION_NAME.docker]: 'docker',
```
`docker.container` metricset doesn't have `cpu`, `memory`, `tx` and `rx`: https://www.elastic.co/guide/en/beats/metricbeat/current/metricbeat-module-docker.html

It's easier to verify if the `docker` object exists in the document